packages/workflow/src/NodeHelpers.ts

```diff
@@ -1181,19 +1303,20 @@ export function mergeIssues(destination: INodeIssues, source: INodeIssues | null
 	}
 }
 
-
-
 /**
  * Merges the given node properties
  *
  * @export
  * @param {INodeProperties[]} mainProperties
  * @param {INodeProperties[]} addProperties
  */
-export function mergeNodeProperties(mainProperties: INodeProperties[], addProperties: INodeProperties[]): void {
+export function mergeNodeProperties(
+	mainProperties: INodeProperties[],
+	addProperties: INodeProperties[],
+): void {
 	let existingIndex: number;
 	for (const property of addProperties) {
-		existingIndex = mainProperties.findIndex(element => element.name === property.name);
+		existingIndex = mainProperties.findIndex((element) => element.name === property.name);
```

Comment:
I honestly prefer it without the extra quotes

Reply:
From Ben: TS recommends this and for consistency we should keep these parens.